vnext/ReactWindowsCore/WinRTWebSocketResource.cpp

```diff
@@ -64,14 +66,16 @@ namespace
 
       void await_suspend(std::experimental::coroutine_handle<> resume)
       {
-        m_queue.Post([context = resume.address()]() noexcept
+        m_callback = [context = resume.address()]() noexcept
```

Comment:
>m_callback [](start = 8, length = 10)

Wasn't this already part of another change?

Reply:
The other change was merged against master. This one want against `0.60-stabe`.
I'll add the comments on the next PR.